web/src/i18n/locales/en_US.ts

```diff
@@ -316,6 +316,13 @@ const translations: VariantResource = {
   Plugin_SheepScanlations_Settings_UsernameInfo: 'Username for automatic account login to Sheep-Scanlations 😉',
   Plugin_SheepScanlations_Settings_Password: 'Password',
   Plugin_SheepScanlations_Settings_PasswordInfo: 'Password for automatic account login to Sheep-Scanlations 😉',
+
+  Plugin_Lezhin_Settings_Username: 'Username',
+  Plugin_Lezhin_Settings_UsernameInfo: 'Username for automatic account login to Lezhin',
```

Comment:
We already have entries for "Username' and 'Password'. There are not generic tho. And they are used for the sample website. Well there is no problem using them anyway.